src/class/gc.rs

```diff
@@ -10,12 +10,16 @@ use std::os::raw::{c_int, c_void};
 pub struct PyTraverseError(c_int);
 
 /// GC support
+#[allow(clippy::upper_case_acronyms)]
```

Comment:
Should this just become a global allow? It is pretty common here (and nothing we can do about it).